deploy/docker-run.sh

```diff
@@ -4,5 +4,5 @@ set -u   # crash on missing env variables
 set -e   # stop on any error
 
 # run uwsgi
-cd /app/
+cd /src/
```

Comment:
In de dockerfile noem je de map `/app`. Dat gaat hier mis. Liefst in Dockerfile aanpassen naar `/src`.